pkg/apis/work/v1alpha1/work_types.go

```diff
@@ -30,6 +30,13 @@ const (
 	ResourcePluralWork = "works"
 	// ResourceNamespaceScopedWork indicates if Work is NamespaceScoped.
 	ResourceNamespaceScopedWork = true
+
+	// OrphaningDeletionPolicyAnnotation is added to resource template annotation to indicates how removal action of managed resources.
+	OrphaningDeletionPolicyAnnotation = "orphaningdeletionpolicy.karmada.io"
+
+	// OrphaningDeletionCompleteAnnotation means that the resource in member clusters completely independent of karmada
+	// Not only cleanup the labels/annotations from the resource on member clusters, but also delete work object in control plane
+	OrphaningDeletionCompleteAnnotation = "complete"
```

Comment:
I raise one question, for the resources that follow the distribution, do we also need to consider, for example, when the deployment is deleted, what is the deletion policy of the following resources such as comfigmap and secret? How do you resolve conflicts when a resource that follows a distribution has its own deletion policy?

Reply:
Another question, when deleting namespace in control plane without OrphaningDeletionPolicyAnnotation , but there  is still resources in namespace in member cluster,  will the namespace in the member clusters be deleted or not?
I think it's better to retain the namespace in member cluster.

Reply:
Your comments will be considered and a new proposal will be submitted for discussion